function_calling/notebooks/function_calling_guide.ipynb

```diff
@@ -9,7 +9,7 @@
   },
```

Comment:
In general I would suggest adding a few markdown lines before every section and/or subsection, to make the notebook more readable and give a general sense of what we aim to do.

Reply:
added extra md cell to the guide notebook